build_tools/azure/install.sh

```diff
@@ -101,7 +101,10 @@ if [[ "$COVERAGE" == "true" ]]; then
 fi
 
 if [[ "$TEST_DOCSTRINGS" == "true" ]]; then
-    python -m pip install sphinx numpydoc  # numpydoc requires sphinx
+    # numpydoc requires sphinx
+    # pin sphinx until jinja2 (>2.10.1) is not released
```

Comment:
until jinja2 (>2.10.1) is released with a fix the collections.abc so as to not raise a spurious deprecation warning